src/stop.sh

```diff
@@ -84,6 +84,15 @@ while [ $# -ge 1 ]; do
 done
 
 if [ $stop_all -eq 1 ]; then
+    #Get fs list from the cluster
+    FSS=$(bin/ceph fs dump --format=json 2> /dev/null | jq -r '.filesystems[].mdsmap.fs_name')
+    #Get mount point for each fs of the cluster
+    CEPH_MNTS=$(for fs in $FSS;do findmnt -n --raw --options=mds_namespace=$fs --output=target;done)
```

Comment:
If the mounted filesystem doesn't specify a mds_namespace, I don't think this will do any good.

IMO, this ought to check whether the mounted monitor addresses/ports correspond to the vstart monitors, and only unmount them if they do. That probably does mean parsing the mount device string though (which is icky, granted).

Reply:
agreed, that's a better fix

Reply:
Done.
@jtlayton @batrick PTAL
Among the list of IP:PORTS from mount, only one IP:PORT (i.e first one) chosen to check if it's present in vstart monitor IP:PORTS list. That's sufficient right?